lib/msal-browser/src/interaction_client/NativeInteractionClient.ts

```diff
@@ -72,6 +93,83 @@ export class NativeInteractionClient extends BaseInteractionClient {
             });
     }
 
+    /**
+     *
+     * @param clientId
+     * @param nativeAccountId
+     * @param request
+     * @param reqTimestamp
+     * @param authority
+     * @returns
+     */
+    protected acquireTokensFromInternalCache(clientId: string, nativeAccountId: string, request: PopupRequest | SilentRequest | SsoSilentRequest, reqTimestamp: number, authority: Authority): AuthenticationResult {
```

Comment:
We should be able to use `SilentCacheClient` for this and provide it with the in memory storage instead of the browser storage object.

Reply:
Not a bad idea, let me check if this optimization can be done.

Reply:
`SilentCacheClient` assumes we use RT to renew tokens. It also reads all tokens from cache, and not just `idToken` and `accessToken`. I would rather keep this simple rather than refactor the code in `SilentCacheClient`.

Reply:
Refactored this code to call `SilentCacheClient` instead.